<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkSettingsStepWidget</class>
 <widget class="QWidget" name="NetworkSettingsStepWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QRadioButton" name="wiredDHCPRadioBtn">
       <property name="text">
        <string>&amp;DHCP</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QRadioButton" name="wiredStaticRadioBtn">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Static</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="ipAddressLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Use the following &amp;IP address:</string>
       </property>
       <property name="buddy">
        <cstring>staticIpAddrlineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="staticIpAddrlineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="subnetMaskLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Subnet &amp;mask:</string>
       </property>
       <property name="buddy">
        <cstring>subnetMaskLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="subnetMaskLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="defGatewayLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Default &amp;Gateway:</string>
       </property>
       <property name="buddy">
        <cstring>defaultGatewayLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="defaultGatewayLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QGroupBox" name="DNSgroupBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>DNS</string>
       </property>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="1" column="0">
         <widget class="QRadioButton" name="dnsYesRadioBtn">
          <property name="text">
           <string>&amp;Yes</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="dnsNoRadioBtn">
          <property name="text">
           <string>N&amp;o</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="dnsServer1Label">
          <property name="text">
           <string>DNS server &amp;1:</string>
          </property>
          <property name="buddy">
           <cstring>dnsServer1LineEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="dnsServer1LineEdit"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="dnsServer2Label">
          <property name="text">
           <string>DNS server &amp;2:</string>
          </property>
          <property name="buddy">
           <cstring>dnsServer2LineEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="dnsServer2LineEdit"/>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Obtain DNS address automatically?</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="text">
        <string>&amp;Back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="text">
        <string>&amp;Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>wiredStaticRadioBtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>staticIpAddrlineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wiredStaticRadioBtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>subnetMaskLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wiredStaticRadioBtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>defaultGatewayLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wiredStaticRadioBtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>DNSgroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wiredStaticRadioBtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>ipAddressLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wiredStaticRadioBtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>subnetMaskLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>41</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wiredStaticRadioBtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>defGatewayLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dnsNoRadioBtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>dnsServer1LineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dnsNoRadioBtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>dnsServer2LineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
